main.c

```diff
@@ -657,7 +657,7 @@ int tcmur_get_time(struct tcmu_device *dev, struct timespec *time)
 
 	ret = clock_gettime(CLOCK_MONOTONIC_COARSE, time);
 	if (!ret) {
-		tcmu_dev_dbg(dev, "Current time %lu secs.\n", time->tv_sec);
```

Comment:
Yeah, this make sense, while we'd better switch to PRIdMAX macro to make it constant with others in tcmu-runner.

Reply:
It's first time I hear about it.
I'll try to do this ;D

Reply:
Fixed!
I edited commit in place with rebase, nothing else changed.